pinot-core/src/test/java/org/apache/pinot/core/data/function/DateTimeFunctionEvaluatorTest.java

```diff
@@ -160,8 +165,32 @@ public void testDateTimeTransformFunctions(String transformFunction, List<String
 
     GenericRow row9_1 = new GenericRow();
     row9_1.putValue("fifteenSecondsSinceEpoch", 80640000L);
-    inputs.add(new Object[]{"toEpochMinutesBucket(fromEpochSecondsBucket(fifteenSecondsSinceEpoch, 15), 10)", Lists.newArrayList(
-        "fifteenSecondsSinceEpoch"), row9_1, 2016000L});
+    inputs.add(
+        new Object[]{"toEpochMinutesBucket(fromEpochSecondsBucket(fifteenSecondsSinceEpoch, 15), 10)", Lists.newArrayList(
+            "fifteenSecondsSinceEpoch"), row9_1, 2016000L});
+
+    // toDateTime simple
+    GenericRow row10_0 = new GenericRow();
+    row10_0.putValue("dateTime", 98697600000L);
+    inputs.add(new Object[]{"toDateTime(dateTime, 'yyyyMMdd')", Lists.newArrayList("dateTime"), row10_0, "19730216"});
+
+    // toDateTime complex
+    GenericRow row10_1 = new GenericRow();
+    row10_1.putValue("dateTime", 1234567890000L);
+    inputs.add(new Object[]{"toDateTime(dateTime, 'MM/yyyy/dd HH:mm:ss')", Lists.newArrayList(
+        "dateTime"), row10_1, "02/2009/13 15:31:30"});
+
+    // toDateTime simple
+    GenericRow row11_0 = new GenericRow();
+    row11_0.putValue("dateTime", "19730216");
+    inputs
+        .add(new Object[]{"fromDateTime(dateTime, 'yyyyMMdd')", Lists.newArrayList("dateTime"), row11_0, 98697600000L});
+
+    // toDateTime complex
```

Comment:
can you add some test cases that deal with time zones and conversion of time zones?

Reply:
btw, the last 3 tests are failing in this file. Check the link below for Travis